packages/jest-serializer-merge-styles/src/index.ts

```diff
@@ -37,38 +37,42 @@ function _serializeRules(rules: string[], indent: (val: string) => string): stri
     const insertedRules = rules[i + 1];
 
```

Comment:
It would be great to have some comments in this file, if not inline comments, then perhaps JSDoc for print() and test().

In particular for the function `test`, it would be great to have some inline comments describing the approach here.

Reply:
Added!